static/app/views/settings/organizationMembers/organizationMembersWrapper.tsx

```diff
@@ -172,6 +175,19 @@ function renderInviteMembersButton({
     >
       {action}
     </Hovercard>
+  ) : disabledBySso ? (
+    <Hovercard
```

Comment:
Should this be a hovercard? Feels like this should just be a tooltip?

Reply:
Sure. I was just imitating the other "disabled" message.

Any further, subjective tweaks to phrasing or presentation are welcome. I was just winging it on the message to the user.